pulseapi/entries/test_views.py

```diff
@@ -358,8 +358,14 @@ def test_bookmarked_entries_view(self):
         bookmarkResponse = self.client.get('/api/pulse/entries/bookmarks/')
         self.assertEqual(bookmarkResponse.status_code, 200)
 
+        # verify data returned has the following properties
         bookmarkJson = json.loads(str(bookmarkResponse.content, 'utf-8'))
-        self.assertEqual(len(bookmarkJson), 1)
+
+        self.assertEqual('count' in bookmarkJson, True)
+        self.assertEqual('previous' in bookmarkJson, True)
+        self.assertEqual('next' in bookmarkJson, True)
+        self.assertEqual('results' in bookmarkJson, True)
+        self.assertEqual(len(bookmarkJson), 4)
```

Comment:
this looks like it's missing the bits to verify that `bookmarkJson.results` still has the correct content